install_config/install/docker_registry.adoc

```diff
@@ -1013,6 +1013,7 @@ storage:
     readonly:
       enabled: false
 ----
+<1> This entry is *mandatory* for image pruning to work properly.
```

Comment:
It wasn't immediately obvious to me. But that's my one and only argument. If you find it sufficient, I won't argue.